libvast/test/system/counter.cpp

```diff
@@ -166,4 +168,76 @@ TEST(count IP point query with partition - local stores) {
   self->send_exit(counter, caf::exit_reason::user_shutdown);
 }
 
+TEST(count meta extractor age 1) {
+  // Create an index with partition-local store backend.
+  auto indexdir = directory / "index2";
+  auto index
+    = self->spawn(system::index, system::accountant_actor{}, fs, archive,
+                  indexdir, "segment-store", defaults::import::table_slice_size,
+                  100, 3, 1, indexdir, 0.01);
+  // Fill the INDEX with 400 rows from the Zeek conn log.
+  auto slices = take(zeek_conn_log_full, 4);
+  for (auto& slice : slices) {
+    slice = slice.unshare();
+    slice.import_time(time::clock::now());
```

Comment:
Do we have to set a valid offset here?

Reply:
Nope, `slice.unshare()` already takes care of that.

Reply:
Only if the slice had a valid offset to begin with, but it doesn't seem to be relevant for this test anyways.